### DISCUSSION TITLE
I would love an option to "Keep current LUT"

### DISCUSSION THREAD

**Discussion by sargentpilcher** [videovillage/community#197] [Ideas]
To be more specific, I often want to use the same LUT on multiple video files, and I would really like an option to just default to a LUT, or keep the same LUT for a playlist, or just until I change it. It's tedious having to pick the same LUT on every video as I'm previewing video files. 

Still better than firing up Davinci Resolve though!

**Comment by Paulexe** [comment]
I need this as well!
